### PR TITLE
HTTP Endpoint for getting unsingned contract txs

### PR DESCRIPTION
**Pull request by velzevur**
This is the first of a series of commits implementing endpoints for getting valid unsinged transactions that can be signed outside the node and posted again.

Also implemented some helper functions for chaining request processing functions.

#### Create contract
Get a semantically correct unsigned `contract_create_tx`
```
POST /v2/tx/contract/create
```
mandatorary params: owner, code, vm_version, deposit, amount, gas, gas_price, fee, call_data
optional params: nonce; if not provided - `owner`-s account next nonce is used.
#### Create contract
Get a semantically correct unsigned `contract_call_tx`
```
POST /v2/tx/contract/call
```
mandatorary params: caller, contract, vm_version, amount, gas, gas_price, fee, call_data
optional params: nonce; if not provided - `caller`-s account next nonce is used.

### CODE REVIEW

File: apps/aecontract/src/aect_ring.erl [hunk 1]
```diff
@@ -53,6 +56,15 @@ hexstring_encode(Code) ->
            || <<X:4, Y:4>> <= Code >>,
     <<"0x", CodeAsHexString/binary >>.
     
+hexstring_decode(Code) ->
+    <<"0x", CodeAsHexString/binary >> = Code,
+    0 = bit_size(CodeAsHexString) rem 8,
```
Comment (hanssv) [inline apps/aecontract/src/aect_ring.erl]: Would it get past the `/binary` at the previous line if it wasn't full bytes?

Comment (velzevur) [reply apps/aecontract/src/aect_ring.erl]:
```
1> Bin = <<"0x">>.
<<"0x">>
2> <<"0x", Rest/binary>> = Bin.
<<"0x">>
3> Rest.
<<>>
```

Comment (velzevur) [reply apps/aecontract/src/aect_ring.erl]: for completeness
```
4> bit_size(Rest).
0
```

File: apps/aecontract/src/aect_ring.erl [hunk 2]
```diff
@@ -53,6 +56,24 @@ hexstring_encode(Code) ->
            || <<X:4, Y:4>> <= Code >>,
     <<"0x", CodeAsHexString/binary >>.
     
+hexstring_decode(Code) ->
+    <<"0x", CodeAsHexString/binary >> = Code,
```
Comment (hanssv) [inline apps/aecontract/src/aect_ring.erl]: This will ensure `CodeAsHexString` is 8*n bits...

**Pull request merged**